test/onnx/dynamo/test_exporter_api.py

```diff
@@ -221,5 +228,94 @@ def test_serialize_succeeds_when_model_greater_than_2gb(self):
             serializer.serialize(onnx_program, io.BytesIO())
 
 
+class TestONNXExportWithDynamo(common_utils.TestCase):
+    def test_args_normalization_with_no_kwargs(self):
+        onnx_program_from_new_exporter = torch.onnx.dynamo_export(
+            SampleModelTwoInputs(), torch.randn(1, 1, 2), torch.randn(1, 1, 2)
+        )
+        onnx_program_from_old_exporter = torch.onnx.export(
+            SampleModelTwoInputs(),
+            (torch.randn(1, 1, 2), torch.randn(1, 1, 2)),
+            dynamo=True,
+        )
+        self.assertEqual(
```

Comment:
Can you say something about how this equality assertion works? Just for my own education, wondering what kind of infrastructure exists for proto-object equality check.

Reply:
That's a good point. I didn't verify that. This might be invalid. I will verify this.

Reply:
There is some internal equality check according to https://proto-plus-python.readthedocs.io/en/stable/_modules/proto/message.html. I think it should work when two messages are _exactly_ the same.

Reply:
It does catch the difference between two protobuf if there is any. Here is their code: https://github.com/pytorch/pytorch/blob/22964d1007d1d87964df1db609da53b35b00316f/torch/testing/_internal/common_utils.py#L3522